app/models/scoreboard.rb

```diff
@@ -1,7 +1,10 @@
-class ScoreboardSetup < ActiveRecord::Base
```

Comment:
I'd have preferred `ScoreboardSetup`. When you say Scoreboard I expect something that saves the scores of the users etc.